corehq/apps/users/templates/users/partials/edit_role_modal.html

```diff
@@ -18,10 +18,12 @@ <h4 class="modal-title" data-bind="text: modalTitle"></h4>
       <div class="modal-body">
         <div class="form form-horizontal">
           <div class="form-group">
-            <label class="col-sm-4 control-label"
-                   for="role-name">
-              {% trans "Role Name*" %}
-            </label>
+            <span class="asteriskField">
```

Comment:
I think Jenny meant for this `span` to encompass only the asterisk, eg:
https://github.com/dimagi/commcare-hq/blob/1cbafaaf72d864a295fb0317b000b44a39c522e2/corehq/motech/dhis2/templates/dhis2/partials/dhis2_dataset_map_templates.html#L17-L17